release-notes.txt

```diff
@@ -1,3 +1,751 @@
++ sys/bitfield: add support for bit-wise bitfield operations (#17710)
++ sys/senml: add SenML modules (#16384)
++ sys/vfs: add vfs_mount_by_path() (#17661)
++ sys/xtimer: introduce `xtimer_is_set()` (#17630)
++ sys/ztimer: add ztimer64_xtimer_compat complete xtimer replace module (#17670)
++ vfs: Introduce reliable disk enumeration (#17660)
+* drivers/dose: make use of ringbuffer for RX (#17210)
+* examples/twr_aloha: refactor (#17406)
+* pkg/fatfs: enable exFAT support (#17798)
+* pkg/fatfs: implement statvfs() (#17634)
+* rust: test on stable (#17805)
+* sc_vfs: print file size (#17622)
+* sys/auto_init: custom auto-initialization sequence (simple) (#17794)
+* sys/event/timeout: remove alternative xtimer implementation (#17689)
+* sys/evtimer: remove deprecated evtimer_now_min (#17655)
+* sys/fido2: use ztimer instead of xtimer (#17753)
+* sys/posix/pthread: newlib compatibility (#17734)
+* sys/test_utils/print_stack_usage: work with small stacks (#17891)
+* sys/vfs: provide vfs_fsync() (#17621)
+* sys/xtimer: make xtimer_ztimer_compat default backend (#17721)
+* sys/ztimer doc: List prerequisites for successful use of ztimer_now (#17614)
+* sys/ztimer64/util.c: fix ztimer64_set_timeout_flag (#17561)
+* sys/ztimer: cleanup xtimer_compat.h for 32Bit only (#17690)
+* sys/ztimer: Name callback types (#17758)
+* sys: some simple xtimer->ztimer conversions (#17892)
+* sys/*timer: rework dependecies to ease backend switch, prefer
+  ztimer_xtimer_compat over xtimer_on_ztimer (#17811)
+* sys/syscalls: make gettimeofday() implementation optional (#17733)
+* sys/ztimer: auto-select ztimer_no_periph_rtt only for samd21 (#17786)
+* sys: sort out ztimer_xtimer_compat and ztimer64_xtimer_compat depes (#17732)
+* tests/pkg_mbedtls: improve entropy test case and doc (#17606)
+* {examples,tests}/rust: Rust updates (-sys/-wrappers version, stable) (#17761)
+
+Networking
+----------
+
++ gcoap: add simple forward-proxy (#13790)
++ gnrc_pktbuf_cmd: add od dependency with gnrc_pktbuf_static (#17228)
++ nanocoap: add nanocoap_get_blockwise_url_to_buf() function (#17833)
++ nanocoap: introduce coap_opt_remove() (#17881)
++ nimble/netif: add support for BT5 PHY modes (#16860)
++ sys,examples,tests,fuzzing: add missing includes (#17714)
++ sys/net/application_layer/sock_dns: add pseudomodule
+  auto_init_sock_dns (#17493)
++ sys/net/application_layer/sock_dns_mock: add module for mocking
+  sock_dns (#17871)
++ sys/net/application_layer: add telnet server module & example (#16723)
++ sys/net/dsm: add missing dependencies (#17685)
++ sys/net/nanocoap: introduce `nanocoap_sock_*()`, use in
+  suit/transport/coap (#17474)
++ sys/net/netutils: add netutils_get_ipv4() (#17764)
++ sys/net/sock: add sock_udp_sendv() API (#17485)
++ sys/net/sock_util: add sock_tl_name2ep() to optionally perform DNS
+  lookups (#17510)
++ sys/stdio_nimble: add new stdio module using nimble (#12012)
+* drivers/atwinc15x0: enable sleep state (#17854)
```

Comment:
Huh this didn't make it [into the release](https://github.com/RIOT-OS/RIOT/commits/2022.04-branch/drivers/atwinc15x0) (and is buggy without #17889)